src/Services/SearchService.php

```diff
@@ -33,6 +34,7 @@ class SearchService implements SearchServiceInterface
     use Loggable;
 
     const DEFAULT_ITEMS_PER_PAGE = 25;
+    const MAX_RETRIES = 2;
```

Comment:
Can we rename this to `MAX_TRIES` and change its value to `3` and make sure exactly `3` tries are being made? Until I saw the test I was assuming that we only do 2 tries.

Reply:
I'm fine with this. I was just wondering if we could improve this somehow, to make it more clear that actually 3 requests are being made in total, but I guess we really need some refactoring for this in the future, as the current state isn't in its best shape.